test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -1325,6 +1325,77 @@ describe('Deployments View widget', () => {
             });
         });
 
+        it('should allow to manipulate workflow parameters', () => {
+            const parameterNames = {
+                notVisible: 'not_visible',
+                visibleAndRequired: 'visible_and_required',
+                visibleAndNotRequired: 'visible_and_not_required'
+            } as const;
+
+            const mockedParameters = {
+                [parameterNames.notVisible]: {
+                    description: 'This option should not be visible within the form',
+                    type: 'node_id',
+                    default: 'test'
+                },
+                [parameterNames.visibleAndRequired]: {
+                    description: 'This option should be visible and required',
+                    type: 'string'
+                },
+                [parameterNames.visibleAndNotRequired]: {
+                    description: 'This option should be visible, but not required',
+                    type: 'string',
+                    default: 'test'
+                }
+            };
+
+            const mockedWorkflow: SimplifiedWorkflow = {
+                name: 'test_workflow',
+                parameters: mockedParameters
+            };
+
+            cy.interceptSp(
+                'POST',
+                {
+                    pathname: '/searches/workflows'
+                },
+                {
+                    items: [mockedWorkflow]
+                }
+            );
+
+            useDeploymentsViewWidget();
+            widgetHeader.openRunWorkflowModal();
+
+            cy.get('.modal').within(() => {
+                cy.getField('Workflow').within(() => {
+                    cy.get('input').click();
+                    cy.get(`div[option-value="${mockedWorkflow.name}"]`).click();
+                });
+
+                cy.getField(parameterNames.notVisible).should('not.exist');
+                cy.getField(parameterNames.visibleAndRequired).should('be.visible');
+                cy.getField(parameterNames.visibleAndNotRequired).should('be.visible');
+
+                cy.clickButton('Run');
+
+                cy.getField(parameterNames.visibleAndRequired).should('have.class', 'error');
+                cy.typeToFieldInput(parameterNames.visibleAndRequired, 'test');
+
+                cy.clickButton('Run');
+
+                cy.interceptSp('POST', '/execution-groups', request => {
+                    const { default_parameters: defaultParameters } = request.body;
+
+                    expect(defaultParameters[parameterNames.notVisible]).to.be.an('undefined');
```

Comment:
Optional:

Very minor. 

I think
```suggestion
                    expect(defaultParameters[parameterNames.notVisible]).to.be('undefined');
```
is more correct from spoken language perspective.

If we had "undefined something", then "an" preposition would make sense here.

Reply:
It was adjusted within dee6c5b44896a0ec3b640708c001d54d6f475943

Reply:
Unfortunately this small change won't be easy to implement

At first I've also tried to go without `an`, but then I've started getting the following error:
```typescript
TypeError: expect(...).to.be is not a function
```

So when it comes to language perspective, I think that `Chai` has it on perspective on that 😄

Reply:
OK, let's leave it then.